src/main/kotlin/racingCar/view/OutputView.kt

```diff
@@ -22,8 +24,10 @@ object OutputView {
     }
 
     fun printWinner(carList: List<Car>) {
-        val cars = Cars()
+        val moveStrategy = RandomStrategy()
+        val cars = Cars(moveStrategy)
         val winners = cars.getWinners(carList)
-        println(winners + WINNER_MESSAGE)
+        val formatWinner = winners.joinToString(COMMA) { it.name }
```

Comment:
참고로 joinToString의 기본 인자가 ", "이므로 생략해도 괜찮습니다!
```suggestion
        val formatWinner = winners.joinToString { it.name }
```